Signal/src/views/OWSAlerts.swift

```diff
@@ -23,15 +23,18 @@ import Foundation
         UIApplication.shared.frontmostViewController?.present(alertController, animated: true, completion: nil)
     }
 
+    public class func showAlert(withTitle title: String) {
+        self.showAlert(withTitle: title, message: nil, buttonTitle: NSLocalizedString("OK", comment: ""))
+    }
+
     public class func showAlert(withTitle title: String, message: String) {
```

Comment:
Well, I've done this but we're almost entirely using this from Objective-C and you can't use default params from Objective-C - at least not to my knowledge.